src/components/organisms/EarthEditor/CanvasArea/convert.ts

```diff
@@ -231,3 +231,55 @@ export const convertToBlocks = (data?: GetBlocksQuery): BlockType[] | undefined
     .filter((a): a is BlockType[] => !!a)
     .reduce((a, b) => [...a, ...b], []);
 };
+
+// TODO: refactor these part since it isn't based on DRY principle
+export const convertLayersWithRawProperty = (
+  data: GetLayersQuery | undefined,
```

Comment:
```suggestion
  data?: GetLayersQuery,
```

Reply:
Similar places in this file also exist so update those as well to keep consistent.